data-pipeline/Cargo.toml

```diff
@@ -16,11 +16,15 @@ log = "0.4"
 rmp-serde = "1.1.1"
 bytes = "1.4"
 tokio = {version = "1.23", features = ["rt"], default-features = false}
+http = "0.2"
 
 ddcommon = { path = "../ddcommon" }
 datadog-trace-protobuf = { path = "../trace-protobuf" }
 datadog-trace-utils = { path = "../trace-utils" }
 datadog-trace-normalization = { path = "../trace-normalization" }
+datadog-trace-obfuscation = { path = "../trace-obfuscation" }
+datadog-ddsketch = { path = "../ddsketch" }
+httpmock = "0.7.0"
```

Comment:
this should either be in dev.dependencies or under a feature like in [trace-utils](https://github.com/DataDog/libdatadog/blob/main/trace-utils/Cargo.toml#L35).